xml_converter/src/attribute/cull_chirality.hpp

```diff
@@ -5,13 +5,15 @@
 
 #include "../rapid_helpers.hpp"
 #include "../rapidxml-1.13/rapidxml.hpp"
+#include "../string_helper.hpp"
 
 using namespace std;
 
-
 class CullChirality {
  public:
-    string chirality;
+    string cull_chirality;
+
+    virtual string classname() { return "CullChirality"; };
 };
 
-CullChirality parse_Chirality(rapidxml::xml_attribute<>* input, vector<XMLError*> *errors);
+CullChirality parse_CullChirality(rapidxml::xml_attribute<>* input, vector<XMLError*> *errors);
```

Comment:
I know I allowed this before because they were a technical requirement for how the , but I have changed my mind and these functions should be named in the same style as other functions. EG:

```c++
CullChirality parse_cull_chirality(rapidxml::xml_attribute<>* input, vector<XMLError*> *errors); 
```